README.md

```diff
@@ -0,0 +1,9 @@
+# Cape Town Testing
+## Home of all things cape town testing related
+
+Thanks for checking out our GitHub page! Here you will find all the resources from previous meetups as well as a few other cool things.
+
+If you would like to know when the next event is happening, please check out our [Meetup.com](https://www.meetup.com/cape-town-testing-meetup/) page.
+
+## Speak at an upcoming event!
+Interested in sharing your knowledge and/or experiences, we're interested in hearing about them! Check out the [Misc](../Misc) page for details
```

Comment:
`Check out the [Misc](../Misc) page for details` 
this misc page goes to a 404, is that on purpose? could we send them to twitter or meetup.com in the meantime?
I'll chuck our lightning talk content up somewhere here too, and you do likewise for the speaker stuff?